src/components/t.jsx

```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types';
+
+import { useT } from '../hooks';
+
+const T = ({ name, ...props }) => useT()(name, props);
```

Comment:
Usage looks a bit weird (`<T name="..." />`) but I think this is definitely the way to go. It indicates that whatever is rendered by this component is a react element (in most cases).

Reply:
same